packages/pytest-simcore/src/pytest_simcore/helpers/faker_factories.py

```diff
@@ -288,6 +289,29 @@ def random_product(
     return data
 
 
+def random_product_price(
+    *, product_name: str, fake: Faker = DEFAULT_FAKER, **overrides
+) -> dict[str, Any]:
+    from simcore_postgres_database.models.products_prices import products_prices
+
```

Comment:
Q: I believe we have models for all these things. Why we do not use them and go with manually created dictionaries? we could leverage the examples and just export as dictionary if that is really what one wants. Just a question, no need to adjust now.

Reply:
https://github.com/ITISFoundation/osparc-simcore/issues/7281